spec.md

```diff
@@ -2153,6 +2158,46 @@ operand (when the stride is 1). By contrast, slicing a list requires
 the creation of a new list and copying of the necessary elements.
 
 
+### Lambda expressions
+
+A `lambda` expression yields a new function value.
+
+```grammar {.good}
+LambdaExpr = 'lambda' [Parameters] ':' Test .
```

Comment:
I'm in a rush so not checking the full grammar atm, but aren't there two cases of the concrete grammar production for lambda, that differ in the expressiveness of the body?

Reply:
Yes, but it's really to enforce the correct precedence of if/else expressions. I've added a note in Conditional Expressions.

Reply:
During parsing, the `if` operator, considered as a postfix operator on                               
the "true" expression, has higher precedence than `else` (a prefix                                   
operator on the "false" expression), which in turn has higher                                        
precedence than the `lambda` prefix operator.                                                        
                                                                                                     
```python                                                                                            
a if b else (c if d else e)          # parens are redundant                                          
(a if b else c) if d else e          # parens are required                                           
                                                                                                     
lambda: (a if b else c)              # parens are redunant                                           
(lambda: a) if b else c              # parens are required                                           
                                                                                                     
a if b else lambda: (c if d else e)  # parens are redundant                                          
a if b else (lambda: c if d else e)  # parens are required                                           
(a if b else lambda: c) if d else e  # parens are required                                           
```